gcp_variant_transforms/beam_io/vcf_header_io.py

```diff
@@ -193,3 +194,86 @@ def __init__(
 
   def expand(self, pvalue):
     return pvalue.pipeline | Read(self._source)
+
+
+class _WriteVcfHeaderFn(beam.DoFn):
+  """A DoFn for writing VCF headers to a file."""
+
+  HEADER_TEMPLATE = '##{}=<{}>\n'
+  FINAL_HEADER_LINE = '#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT\n'
+  INFO_KEY = 'INFO'
+  FILTER_KEY = 'FILTER'
+  ALT_KEY = 'ALT'
+  FORMAT_KEY = 'FORMAT'
+  CONTIG_KEY = 'contig'
+  ID_KEY = 'ID'
+  NUMBER_KEY = 'Number'
+  TYPE_KEY = 'Type'
+  DESCRIPTION_KEY = 'Description'
+
+  def __init__(self, file_path):
+    self._file_path = file_path
+
+  def process(self, header):
+    with FileSystems.create(self._file_path) as file_to_write:
+      self._write_headers_by_key(file_to_write, self.INFO_KEY, header.infos)
```

Comment:
I think it's better to treat high-level "key"s as "types" instead (there are too many keys in the header spec, so it's difficult to follow which is which). So, this would be _write_headers_by_type

Reply:
Done.